contents/customers/contra.md

```diff
@@ -0,0 +1,54 @@
+---
+title: How Contra used session recordings to improve registrations by 30%
+customer: Contra
+logo: ../images/customers/contra/contra_logo.png
+featuredImage: ../images/customers/contra/contra_featured.png
+industries:
+    - Saas
+users:
+    - Product
+    - Engineering
+toolsUsed:
+    - Feature Flags
+    - Session Recording
+    - Product Analytics
+---
+
+Contra is a freelance marketplace for service independent creatives and engineers. It empowers its community to find work that fits the way they want to live. Since 2022 [Contra](http://www.contra.com), has used PostHog Cloud to power many of its engineering and product decisions, though previously the stack was a lot more complex.
+
+“Originally we were using multiple platforms,” said Allison Nulty, Head of Product at Contra. “That meant Mixpanel for analytics, LaunchDarkly for feature flags, Segment for front-end tracking and FullStory for recordings. In isolation they could work well, but they were costly even then.”
+
+As a result of spreading functionality across multiple tools, Contra also faced data inconsistencies. Tools such as Segment were helpful for forwarding data between platforms, but also added to the costs and upkeep required. Eventually, Contra decided to consolidate the stack as much as possible — and that’s where PostHog came in. 
+
+### Using feature flags for a ‘slow roll-out’ strategy
+
+Switching to PostHog enabled Contra to consolidate feature flags, session recordings, product analytics and more into a single tool — but it also enabled the company to come together around a single platform as well. Instead of each team in the company relying on a different software, everyone was able to look at a single source of truth.
+
+<BorderWrapper>
+    <Quote
+        imageSource="/images/customers/allison.jpg"
+        size="md"
+        name="Allison Nulty"
+        title="Head of Product, Contra "
+        quote={`“A real positive has been the ability to talk to the PostHog engineers, to share feedback with them and have it received well. I think how rewarding that open line of communication is when I engage with our users too.”`}
```

Comment:
Updated (to sound better): “A huge competitive advantage has been the ability to talk directly with the PostHog technical team via Slack. We can actively share feedback, ask questions, and make feature requests  and always see a quick turnaround time and thoughtful suggestions. At Contra, we apply these same principles of collaborating with our community to build our product and so we appreciate that level of service when on the other side.

Reply:
```suggestion
        quote={`“A huge competitive advantage has been the ability to talk directly with PostHog engineers over Slack. We share feedback, ask questions, and make requests and always see a quick response time and thoughtful suggestions. At Contra, we apply these same principles of collaborating with our community to build our product.”`}
```